### PR TITLE
Sambamba/samtools disconnect and Sambamba 0.5.9

### PR DESCRIPTION
**Pull request by vinjana**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by bgruening** [comment]
@vinjana what about leaving 0.6.6 as it is and creating 0.6.7 without samtools as dependency?

**Comment by vinjana** [comment]
You mean 0.5.9 and 0.6.7 without the samtools dependency and leave 0.6.6 as is? I can do that. Just validate that this is what you meant.

File: recipes/sambamba/meta.yaml [hunk 1]
```diff
@@ -11,13 +11,7 @@ source:
   md5: 07d82a2102126565e6796658face7c71 # [osx]
 
 build:
-  number: 0
-
-requirements:
-  build:
-  run:
-    - samtools # required for mpileup
-    - bcftools # required for mpileup
```
Comment (johanneskoester) [inline recipes/sambamba/meta.yaml]: Does that mean those are no longer needed by sambamba?

Comment (vinjana) [reply recipes/sambamba/meta.yaml]: No. It means they were never really needed by sambamba (depending on how you look at it :-) ). They are only needed for sambamba mpileup, whose help-page also says so. Please see the discussion at  #6974.

**Comment by vinjana** [comment]
O.k. how to proceed here? Acceptible?

**Comment by johanneskoester** [comment]
Thanks!

**Pull request merged**